File: ABC/abc025/abc025_b.py
N, A, B = map(int, input().split())

ans = 0
for _ in range(N):
    s, d = input().split()
    d = int(d)

    d = max(d, A)
    d = min(d, B)
    
    if s == "West":
        d *= -1
    
    ans += d

if ans > 0:
    print("East", end=" ")
elif ans < 0:
    print("West", end=" ")

print(abs(ans))

File: ABC/abc025/abc025_b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int N, A, B, d, ans = 0;
    string s;

    cin >> N >> A >> B;
    for (int i = 0; i < N; i++) {
        cin >> s >> d;

        d = max(d, A);
        d = min(d, B);

        if (s == "West") d *= -1;

        ans += d;
    }

    if (ans > 0) cout << "East ";
    else if (ans < 0) cout << "West ";

    cout << abs(ans) << endl;
}